src/google_cloud_sdk_emulators/mod.rs

```diff
@@ -1,19 +1,21 @@
 use testcontainers::{core::WaitFor, Image, ImageArgs};
 
-const NAME: &str = "google/cloud-sdk";
-const TAG: &str = "362.0.0-emulators";
+const NAME: &str = "gcr.io/google.com/cloudsdktool/google-cloud-cli";
+const TAG: &str = "471.0.0-emulators";
 
 const HOST: &str = "0.0.0.0";
 pub const BIGTABLE_PORT: u16 = 8086;
 pub const DATASTORE_PORT: u16 = 8081;
 pub const FIRESTORE_PORT: u16 = 8080;
 pub const PUBSUB_PORT: u16 = 8085;
-pub const SPANNER_PORT: u16 = 9010;
+pub const SPANNER_GRPC_PORT: u16 = 9010;
+pub const SPANNER_REST_PORT: u16 = 9020;
```

Comment:
I can leave `SPANNER_PORT` unchanged (or marked as `deprecated`), but a breaking change is unavoidable since we also need to change `pub struct CloudSdkArgs` [here](https://github.com/testcontainers/testcontainers-rs-modules-community/pull/117/files#diff-67979b0db9e7a8cc4a40750c2b578b26481867de29a9bcdeb37efa18514f44d9R18). Presumably, this is less likely to break people, but it's still a breaking API change.

Do we care about that other breaking change?

Reply:
Well, in that case I think we can go only with breaking change flag.
Thus, I think it's ok to bump the version as well. But better to preserve the registry as dockerhub


I'd suggest to refactor `CloudSdkArgs` a bit to avoid such breaking changes in the future, e.g: make the fields private and expose builder-like functions. So we can add new arguments in the future without breaking changes, just by adding `.with_my_arg(...)` (just for example)

Reply:
Yeah, that makes sense. I'll take a stab at this later today.

Reply:
Hi there!

Any progress? We're going to prepare a new release with breaking changes soon. 
So would be nice to incorporate this one as well

Reply:
We could go with 2 `Vec<String>`,  but accept something like `&[impl Into<String>]`.
This way, users can define their own structures which implement `Display` (e.g enums).

For test scenarios it's usually ok to hardcode some values or create simple wrappers.
Otherwise it can be too complicated to support all possible arguments. It could lead to a mess (especially across different versions)

For example, ENV variables also could be represented as typed structs, but it's too complicated for testing purposes. And additional complexity for maintainers